src/app/layout.tsx

```diff
@@ -29,6 +30,9 @@ export async function generateMetadata(): Promise<Metadata> {
       url: "https://boss.community",
       images: ["https://boss.community/images/BUILD-thumbnail.jpg"],
     },
+    other: {
+      ...(await fetchMetadata(new URL(FRAMES_BASE_PATH, appURL()))),
+    },
```

Comment:
Can you explain for my benefit why this is needed? 
Adding this causes every request to every page to trigger a request to /frames, which leads to quite a bit of noise. Are you aware of a way to avoid this?

Reply:
it could be moved to a specific page instead of the layout.tsx file